upload/extension/opencart/admin/controller/payment/opayo.php

```diff
@@ -0,0 +1,351 @@
+<?php
+/**
+ * Class Opayo
+ *
+ * @package Opencart\Admin\Controller\Extension\Opencart\Payment
+ */
+namespace Opencart\Admin\Controller\Extension\Opayo\Payment;
+class Opayo extends \Opencart\System\Engine\Controller {
+	/**
+	 * @var array
+	 * @var string
+	 */
+	private array $error = [];
+	private string $separator = '';
+
+	/**
+	 * Constructor
+	 *
+	 * @param \Opencart\System\Engine\Registry $registry
+	 */
+	public function __construct(\Opencart\System\Engine\Registry $registry) {
+		parent::__construct($registry);
+
+		if (VERSION >= '4.0.2.0') {
+			$this->separator = '.';
+		} else {
+			$this->separator = '|';
+		}
+	}
```

Comment:
Since this is being merged in to OC 4.0.2.3 + 1 all of this can be simplified as:
```suggestion
	private string $separator = '.';
```

Reply:
Yeah, not sure why extension developers keeps putting that one in. I'll remove it.